serveradmin/serverdb/query_materializer.py

```diff
@@ -197,62 +196,59 @@ def _add_domain_attribute(self, attribute, servers):
             )
 
     def _add_supernet_attribute(self, attribute: Attribute, servers_in):
-        if attribute.inet_address_family:
-            self._add_supernet_attribute_af(attribute, servers_in)
-        else:
-            self._add_supernet_attribute_intern_ip(attribute, servers_in)
-
-    def _add_supernet_attribute_af(self, attribute, servers_in):
-        supernet_id = ServerInetAttribute.objects.filter(
-            value__net_contains_or_equals=OuterRef("value"),
-            server__servertype_id=attribute.target_servertype_id,
-            attribute__inet_address_family=attribute.inet_address_family,
-        ).values("server__server_id")
-
-        attrs = ServerInetAttribute.objects.filter(
-            server_id__in=[s.server_id for s in servers_in],
-            attribute__inet_address_family=attribute.inet_address_family,
-        ).annotate(
-            supernet_id=Subquery(supernet_id, output_field=AutoField()),
-        )
-
-        supernets = Server.objects.filter(server_id__in=attrs.values("supernet_id"))
-        supernets_by_id = {s.server_id: s for s in supernets}
-
-        for a in attrs:
-            if a.supernet_id in supernets_by_id:
-                self._server_attributes[a.server][attribute] = supernets_by_id[
-                    a.supernet_id
-                ]
+        q = """
```

Comment:
Care to share what this query is doing and why we by pass the query set interface in a comment ?

Reply:
It's not a comment, it's a multiline string.

Reply:
Yes I mean maybe you can describe the query's intention in a comment not everybody might look at the SQL and find it obvious what it is doing.

Reply:
And more importantly why.

Reply:
Done.